### ISSUE TITLE
Automate updating database with new batch

### ISSUE THREAD

**Issue by jasonaowen** [jasonaowen/recurse-faces#27]
The database needs to be kept up-to-date as new batches start and residents join. Right now, populating the database with past & present Recursers is a manual process.

Add some method to use the RC API auth token granted during login (which is currently being verified & discarded) to check for new people. This could be a protected button that only some people can see, but probably we could get away with having a confirmation dialog saying "please don't press this if you don't think it needs pressing" or the like.

**Comment by jasonaowen** [comment]
One possible workflow:

1. Press button
2. Set cookie ("requesting db update")
3. Do auth flow with RC
4. On callback, if cookie is present, make additional API calls with the just-obtained auth token

**Comment by jasonaowen** [comment]
Now that the RC API has personal access tokens, I rewrote the update data script in eb62662b8c42578249de912f34fc984e0c3c1715 to get everything from the new paged profiles API. I configured a daily job with the [Heroku scheduler add-on](https://devcenter.heroku.com/articles/scheduler) to run at midnight EDT (04:00 UTC), since they don't have a weekly job; it takes less than a minute from start to finish.

I'm super excited to always have up-to-date profiles on Faces and never have to manually update it again! :tada:
